po/nl_NL.po

```diff
@@ -0,0 +1,513 @@
+# French translations for oomox package
```

Comment:
i don't think you should still keep this copyright paragraph here

Reply:
You're right, I'll change that. Thanks for the heads-up!